### PR TITLE
Add HADES FCS in Made with KiCad

### PR DESCRIPTION
**Pull request by pms67**
Added my project to the 'Made With KiCad' section. All hardware was designed in KiCad.

Open-source flight control system for unmanned aerial vehicles, with all components designed completely from scratch. Further details: https://github.com/pms67/HadesFCS

### CODE REVIEW

**Comment by marekr** [comment]
Can you downsize your image file? It's 1.69MB...

**Comment by pms67** [comment]
I downsized the image, should be much smaller now - apologies!

**Comment by nickoe** [comment]
@pms67 Sorry for not responding earlier, but I wanted to look at it myself, but realize I don't have time right now. But the image is still quite big in file size. I think you can easily half the pixel size and maybe add some compression to the jpeg. Hopefully the there are full resolution images in your project repo.

It would be nice to remove the shadow around the board as well I think, this should also be able to take a bit off the filesize in the compression I think.

**Comment by pms67** [comment]
@nickoe No problem - I have removed the shadow around the board, decreased the resolution, and finally increased the JPG compression. (Let me know if it looks too 'crummy' now for the site.)
This version is around 70kB.

File: content/made-with-kicad/hades-fcs.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
++++
+title = "HADES Flight Control System"
+projectimage = "/img/madewithkicad/hades-fcs.jpg"
+projectdeveloper = "Philip Salmony"
+projecturl = "https://github.com/pms67/HadesFCS/"
++++
+
+link:https://github.com/pms67/HadesFCS[HADES FCS] is an open-source flight control system for unmanned aerial vehicles (UAVs) designed completely from scratch. This includes the hardware (4-layer PCBs designed with KiCAD), STM32 firmware (using FreeRTOS), Kalman filtering for state estimation, control algorithms, base station, communication protocol, flight simulator, signal processing, and much, much more.
```
Comment (nickoe) [inline content/made-with-kicad/hades-fcs.adoc:8]: Maybe just strp `s designed with KiCAD` out as this is intended to go on a website where the boards are only made with kicad.

Comment (pms67) [reply content/made-with-kicad/hades-fcs.adoc]: Okay, cool thank you for taking the time to look through this!
I've changed the description and removed the 'designed with KiCad' part. I hope everything should be okay now!

**Comment by nickoe** [comment]
@pms67 Thank you very much :)

**Pull request merged**